mysql-test/main/start_slave_until.test

```diff
@@ -0,0 +1,43 @@
+# ==== Purpose ====
+#
+# Verify that START SLAVE UNTIL replicates correctly when valid parameters are passed,
+# and that it returns errors when invalid combinations of parameters are used.
+#
+# ==== Method ====
+#
+# On master, create tables and insert rows to generate binlog events.
+# Test START SLAVE UNTIL on the slave with:
+#   Incorrect parameter combinations to verify that errors are raised.
+#
+# ==== Related bugs ====
+# MDEV-35635
+# Testing issues around invalid parameters and START SLAVE UNTIL behavior.
+
+# Include necessary configurations
```

Comment:
You shouldn't need any pre-amble `source` for this test, as it is just testing parsing. I think really all that this test needs are the actual `start slave` lines, no set-up data is needed. Then, for a "passing" test, you can just set the expectation `ER_BAD_SLAVE` (as no slave is actually configured, but it proves the parsing has gone through)